app/assets/stylesheets/base.css.scss

```diff
@@ -275,6 +275,10 @@ fieldset.form-group {
   margin-bottom: 15px;
 }
 
+.btn.disabled {
+  pointer-events: auto;
+}
+
```

Comment:
This change comes unexpected. I haven't executed the code myself, but was wondering why this is needed now? We are not adding the first disabled button, are we?

Reply:
@kkoehn added it.

But I think the reason is that with this solution, the default `disabled` prevents the tooltip from working and by overwriting it like that, it allows the tooltip to work. But this is just a guess :)

Reply:
Yes it enables the tooltip for disabled buttons without having to wrap the button in a separate div

Reply:
Ah, I see - thanks!

While I am fine with the change itself, I am still concerned about consistency: Shall we go with the previous approach or align the other buttons, too?

Reply:
We reverted back to the previous approach :)